src/server/scripts/Kalimdor/zone_felwood.cpp

```diff
@@ -100,7 +100,58 @@ class npcs_riverbreeze_and_silversky : public CreatureScript
     }
 };
 
+/*######
+## at_ancient_leaf
+######*/
+
+enum AncientMisc
+{
+    QUEST_ANCIENT_LEAF = 7632,
+    NPC_VARTRUS = 14524,
+    NPC_STOMA = 14525,
+    NPC_HASTAT = 14526,
+    MAX_ANCIENTS = 3,
+};
+
+struct AncientSpawn
+{
+    uint32 entry;
+    float fX, fY, fZ, fO;
+};
+
+static const AncientSpawn SpawnLocations[MAX_ANCIENTS] =
+{
+    { NPC_VARTRUS, 6204.051758f, -1172.575684f, 370.079224f, 0.86052f },  // Vartus the Ancient
+    { NPC_STOMA,   6246.953613f, -1155.985718f, 366.182953f, 2.90269f },  // Stoma the Ancient
+    { NPC_HASTAT,  6193.449219f, -1137.834106f, 366.260529f, 5.77332f },  // Hastat the Ancient
+};
```

Comment:
SummonerType is wrong, should be 2 (SUMMONER_TYPE_MAP)
http://collab.kpsn.org/display/tc/creature_summon_groups#creature_summon_groups-summonerType